google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -135,70 +135,104 @@ public static Builder newBuilder() {
 
-      Bucket.IamConfiguration.BucketPolicyOnly bucketPolicyOnly =
-          new Bucket.IamConfiguration.BucketPolicyOnly();
-      bucketPolicyOnly.setEnabled(isBucketPolicyOnlyEnabled);
-      bucketPolicyOnly.setLockedTime(
-          bucketPolicyOnlyLockedTime == null ? null : new DateTime(bucketPolicyOnlyLockedTime));
+      Bucket.IamConfiguration.UniformBucketLevelAccess uniformBucketLevelAccess =
+          new Bucket.IamConfiguration.UniformBucketLevelAccess();
+      uniformBucketLevelAccess.setEnabled(isUniformBucketLevelAccessEnabled);
+      uniformBucketLevelAccess.setLockedTime(
+          uniformBucketLevelAccess == null ? null : new DateTime(uniformBucketLevelAccessLockedTime));
 
-      iamConfiguration.setBucketPolicyOnly(bucketPolicyOnly);
+      iamConfiguration.setUniformBucketLevelAccess(uniformBucketLevelAccess);
 
       return iamConfiguration;
     }
 
     static IamConfiguration fromPb(Bucket.IamConfiguration iamConfiguration) {
-      Bucket.IamConfiguration.BucketPolicyOnly bucketPolicyOnly =
-          iamConfiguration.getBucketPolicyOnly();
-      DateTime lockedTime = bucketPolicyOnly.getLockedTime();
+      Bucket.IamConfiguration.UniformBucketLevelAccess uniformBucketLevelAccess =
+          iamConfiguration.getUniformBucketLevelAccess();
+      DateTime lockedTime = uniformBucketLevelAccess.getLockedTime();
 
       return newBuilder()
-          .setIsBucketPolicyOnlyEnabled(bucketPolicyOnly.getEnabled())
-          .setBucketPolicyOnlyLockedTime(lockedTime == null ? null : lockedTime.getValue())
+          .setIsUniformBucketLevelAccessEnabled(uniformBucketLevelAccess.getEnabled())
+          .setUniformBucketLevelAccessLockedTime(lockedTime == null ? null : lockedTime.getValue())
           .build();
     }
 
     /** Builder for {@code IamConfiguration} */
     public static class Builder {
-      private Boolean isBucketPolicyOnlyEnabled;
-      private Long bucketPolicyOnlyLockedTime;
+      private Boolean isUniformBucketLevelAccessEnabled;
+      private Long uniformBucketLevelAccessLockedTime;
+
+      /**
+       * Deprecated in favor of setIsUniformBucketLevelAccessEnabled().
+       */
+      @Deprecated
+      public Builder setIsBucketPolicyOnlyEnabled(Boolean isBucketPolicyOnlyEnabled) {
+        this.isUniformBucketLevelAccessEnabled = isBucketPolicyOnlyEnabled;
+        return this;
+      }
+
+      /**
+       * Deprecated in favor of setUniformBucketLevelAccessLockedTime().
+       */
+      @Deprecated
+      Builder setBucketPolicyOnlyLockedTime(Long bucketPolicyOnlyLockedTime) {
+        this.uniformBucketLevelAccessLockedTime = bucketPolicyOnlyLockedTime;
+        return this;
+      }
 
       /**
-       * Sets whether BucketPolicyOnly is enabled for this bucket. When this is enabled, access to
+       * Sets whether Uniform Bucket Level Access is enabled for this bucket. When this is enabled, access to
```

Comment:
IIUC the name is not trade marked, and is a descriptive name. So uniform bucket-level access?